etl/steps/data/garden/smoking/2024-05-30/cigarette_sales.meta.yml

```diff
@@ -0,0 +1,115 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Smoking
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  cigarette_sales:
+    variables:
+      manufactured_cigarettes:
+         title: Annual sales of manufactured cigarettes
+         unit: cigarettes
+         short_unit: cigarettes
```

Comment:
this also affects the way it is presented in the chart (on the axis), right? Because that is the reason I kept it as a short unit - I think it could be confusing if the axis did not have that label as I would maybe assume volume of cigarettes sold to be in packs/ per weight than actual cigarettes. Or does it then default to "unit" in the chart?

(This is all theoretical as we are not using this indicator in a graph as of now but also for the future.)